src/quilla/__init__.py

```diff
@@ -48,25 +55,32 @@ def make_parser() -> argparse.ArgumentParser:  # pragma: no cover
         'json',
         help='The json file name or raw json string',
     )
-    parser.add_argument(
-        '--debug',
-        action='store_true',
-        help='Enable debug mode',
+
+    config_group = parser.add_argument_group(title='Configuration options')
+    config_group.add_argument(
+        '-i',
+        '--run-id',
+        action='store',
+        metavar='run_id',
+        default=None,
+        help='A run ID for quilla, if manually passed in.'
+        'Used to set many quilla tests to have the same run ID'
```

Comment:
why do different tests need to share the same run id especially if they are all VP tests?

Reply:
They don’t necessarily need to, this just allows them to if that is how people would like to organize it. The run ID doesn’t just apply to VP tests- it is also included in all report summaries to let users store the logs by run ID as well.

Grouping them by run ID (multiple test files having the same ID) would also allow a full suite of VP tests to be stored together, which means that you can track when multiple tests of the same suite have failed or changed.

Reply:
I’m not certain what the problems could be- if all tests are stored as “<run_id>_<baseline_name>_treatment.png”, for example, they could keep every single treatment image in the same directory and not have any name conflicts. 

It is assumed that all baselines have unique IDs as well, so if the conflict can happen due to multiple tests having the same baseline name for different VisualParity validations, it’ll be happening to the baselines also